airflow/providers/amazon/aws/triggers/glue.py

```diff
@@ -59,5 +59,5 @@ def serialize(self) -> tuple[str, dict[str, Any]]:
 
     async def run(self) -> AsyncIterator[TriggerEvent]:
         hook = GlueJobHook(aws_conn_id=self.aws_conn_id)
-        await hook.async_job_completion(self.job_name, self.run_id, self.verbose)
-        yield TriggerEvent({"status": "success", "message": "Job done"})
+        glue_job_run = await hook.async_job_completion(self.job_name, self.run_id, self.verbose)
+        yield TriggerEvent({"status": "success", "message": "Job done", "value": glue_job_run["JobRunId"]})
```

Comment:
Isn't the `glue_job_run["JobRunId"]` the same as `self.run_id`? 

Perhaps we could just return that, so no need to fetch the value from the dictionary key.
Upto if you'd like to consider the suggestion :)

Reply:
hmm yes you're right :p